code/game/machinery/washing_machine.dm

```diff
@@ -52,15 +52,15 @@
 
 
 	if(crayon)
-		var/_color
+		var/item_color

```

Comment:
eh in this case it's fine, same for the chicken and mice, mostly wanted to point out every unrelated thing changed

the canister is an issue though